types/views/views.go

```diff
@@ -181,3 +181,80 @@ func (v IPPrefixSlice) MarshalJSON() ([]byte, error) {
 func (v *IPPrefixSlice) UnmarshalJSON(b []byte) error {
 	return v.ж.UnmarshalJSON(b)
 }
+
+// MapOf returns a read-only view over m.
+func MapOf[K comparable, V any](m map[K]V) Map[K, V] {
+	return Map[K, V]{m}
+}
+
+// Map is a read-only accessor over a map.
+type Map[K comparable, V any] struct {
+	// It is named distinctively to make you think of how dangerous it is to escape
```

Comment:
This seems to be the old comment. Didn't we flesh it out elsewhere?

Reply:
Oh yeah, I'll update this file too. We updated the generator, these are still manually generated